belvedere-core/src/main/java/zendesk/belvedere/Belvedere.java

```diff
@@ -19,31 +19,30 @@
 @SuppressWarnings("unused")
 public class Belvedere implements InstanceBuilder {
 
-    private final static String LOG_TAG = "Belvedere";
+    final static String LOG_TAG = "Belvedere";
 
     @SuppressLint("StaticFieldLeak")
     private static Belvedere instance;
 
     private final Context context;
-    private final Logger log;
     private final boolean debug;
-    private final String directoryName;
 
     private Storage storage;
     private IntentRegistry intentRegistry;
     private MediaSource mediaSource;
 
     Belvedere(Builder builder) {
         this.context = builder.context;
-        this.log = builder.logger;
+
+        builder.logger.setLoggable(true);
```

Comment:
Fixed. 
The whole logger concept is still a bit odd, I'll look into fixing it a bit later.